{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,60 @@
+## For the Submitter
```

Comment:
I added a comment about this information on this PR: https://github.com/edx/ux-pattern-library/pull/306

My general feeling is that these first couple of paragraphs belong in a CONTRIBUTING.md document rather in the pull request template itself.